setup.sh

```diff
@@ -0,0 +1,100 @@
+#/usr/bin/env bash
+
+test -r ~/.dotfiles_initialized && echo "dotfiles already setup; ~/.dotfiles_initialized exists!" && exit 0
+
+read -p "Setup requires an internet connection, is long, and requires checkins. Is this okay? (Y/n)? " answer
+case ${answer:0:1} in
+    y|Y )
+    ;;
+    * )
+	    exit 0
+    ;;
+esac
+
+set -x
+find ~ -mindepth 1 -maxdepth 1 -type d -exec rmdir {} \;
+# Setup home directories
+test ! -d ~/downloads && test -d ~/Downloads && mv ~/Downloads ~/downloads
+mkdir -p ~/downloads
+
+xdg-user-dirs-update --set DOWNLOAD ~/downloads
+
+mkdir -p ~/scripts
+mkdir -p ~/projects/c
+mkdir -p ~/projects/go
+mkdir -p ~/projects/go/src
+mkdir -p ~/projects/go/pkg
+mkdir -p ~/projects/go/bin
+mkdir -p ~/projects/python
+
+# blow away default
+rm ~/.bashrc
+ln -s `pwd`/home/zaq/.bashrc ~/.bashrc
+# needed for PATH and GO env vars
+set +x
+source ~/.bashrc
+set -x
+
+# Get Applications
+if ! [ -x "$(command -v go)" ]; then
+	echo "===== Downloading latest GoLang ====="
+	curl `curl -s -L https://golang.org/dl | grep 'download downloadBox.\+linux-amd64' | cut -d'"' -f 4` > ~/downloads/golang.tar.gz
+	sudo tar -c /usr/local/ ~/downloads/golang.tar.gz
```

Comment:
Does this work? It is not working on my machine. 

`man tar` says that `-c` is `create a new archive`.

What works for me is:
```
sudo tar -C /usr/local/ -xf ~/downloads/golang.tar.gz
```

Different tar?